client/app/bundles/course/statistics/pages/StatisticsIndex/assessments/AssessmentsScoreSummaryDownload.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react';
+import { defineMessages } from 'react-intl';
+import { Button, Typography } from '@mui/material';
+import { AxiosError } from 'axios';
+import { JobCompleted, JobErrored } from 'types/jobs';
+
+import { downloadScoreSummary } from 'course/statistics/operations';
+import { CourseAssessment } from 'course/statistics/types';
+import Prompt, { PromptText } from 'lib/components/core/dialogs/Prompt';
+import loadingToast, { LoadingToast } from 'lib/hooks/toast/loadingToast';
+import useTranslation from 'lib/hooks/useTranslation';
+
+interface AssessmentsScoreSummaryDownloadProps {
+  assessments: CourseAssessment[];
+}
+
+const translations = defineMessages({
+  selectedNUsers: {
+    id: 'course.statistics.StatisticsIndex.assessments.selectedNUsers',
+    defaultMessage:
+      'Download Score Summary ({n, plural, =1 {# assessment} other {# assessments}})',
+  },
+  download: {
+    id: 'course.statistics.StatisticsIndex.assessments.downloadCsv',
+    defaultMessage: 'Download',
+  },
+  downloadCsvDialogTitle: {
+    id: 'course.statistics.StatisticsIndex.assessments.downloadCsv',
+    defaultMessage: 'Download Score Summary for the following assessments?',
```

Comment:
Nit: Title Case or lower case, make it consistent

Reply:
changed into `Download Score Summary for the following Assessments?`

Reply:
As in, compared to the other translations, I see you use "score summary" at some places and "Score Summary" here.

Reply:
it's because the title case is for title, but the lower case is for toast. They have different way of formatting